src/debug/HyprCtl.cpp

```diff
@@ -1753,7 +1755,6 @@ void runWritingDebugLogThread(const int conn) {
     std::thread([conn]() {
         while (Debug::RollingLogFollow::Get().IsRunning()) {
             if (Debug::RollingLogFollow::Get().isEmpty(conn)) {
-                std::this_thread::sleep_for(1000ms);
```

Comment:
this change creates unnecessary 100% CPU load when no logs and thread is activated by os scheduler

if u decided to make configurable sleep, better to do it configurable in both hyprctl and core. and leave the default settings as they are, please

Reply:
a 1ms sleep should be good enough here, right? I don't see a point in making both of them configurable

Reply:
not exactly. quite a lot of users, including me, using linux to reanimate old machines with low capabilities

i think here we can sleep for similar to code bellow time.

16.(6) ms is nice speep time (60 checks in second)

but it's better to make it customizable imho

Reply:
in this loop we block 2 mutexes, which slows down the entire system (logs can be anywhere)

Therefore, ideally, it is necessary to adhere to the idea that the default cycle repetition periods should be sufficiently large.

Reply:
how about now